spec/amqp0-9-1.stripped.extended.xml

```diff
@@ -191,11 +185,22 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
       <chassis name="server" implement="MUST"/>
     </method>
     <method name="blocked" index="60">
-      <chassis name="server" implement="MAY"/>
-      <field name="reason" type="shortstr"/>
+      <chassis name="server" implement="MUST"/>
+      <chassis name="client" implement="MUST"/>
+      <field name="reason" domain="shortstr"/>
     </method>
     <method name="unblocked" index="61">
-      <chassis name="server" implement="MAY"/>
+      <chassis name="server" implement="MUST"/>
+      <chassis name="client" implement="MUST"/>
+    </method>
+    <method name="update-secret" synchronous="1" index="70">
+      <chassis name="client" implement="MUST"/>
+      <response name="update-secret-ok"/>
+      <field name="new-secret" domain="longstr"/>
+      <field name="reason" domain="shortstr"/>
+    </method>
+    <method name="update-secret-ok" synchronous="1" index="71">
+      <chassis name="server" implement="MUST"/>
```

Comment:
I'm just curious where these updates came from. I'm assuming the definitive spec is in one of our RabbitMQ org repos, eh?

Reply:
Yes, indeed, we have it in our website 😄 
https://rabbitmq.com/resources/specs/amqp0-9-1.extended.xml

The "stripped" version (no comments, more compact) was missing that method and the `312 no-route` error. See https://github.com/rabbitmq/rabbitmq-website/pull/1464

There's also the spec in [amqp.org](https://www.amqp.org/resources/download), but they seem identical to the ones in our website.

Reply:
Well done.